js/bucket.js

```diff
@@ -112,7 +112,7 @@ function trendFn(start, end, interval, overallTarget) {
   const slope = overallTarget / totalIntervals;
 
   // @return {x: String, y: Integer} target point
-  return function(date) {
+  return function (date) {
```

Comment:
This is [new default behavior](https://prettier.io/blog/2020/03/21/2.0.0.html#always-add-a-space-after-the-function-keyword-3903httpsgithubcomprettierprettierpull3903-by-j-f1httpsgithubcomj-f1-josephfrazierhttpsgithubcomjosephfrazier-sosukesuzukihttpsgithubcomsosukesuzuki-thorn0httpsgithubcomthorn0-7516httpsgithubcomprettierprettierpull7516-by-bakkothttpsgithubcombakkot).